website/docs/r/vapp.html.markdown

```diff
@@ -68,6 +68,7 @@ resource "vcd_vapp_vm" "web2" {
 ## Example of vApp with single VM
 
 **Not recommended in v2.0+** : in the earlier version of the provider it was possible to define a vApp with a single VM in one resource, but it is not recommended as of *v2.0+* provider. Please define vApp and VM in separate resources instead.
+**Deprecated in 2.5**: The implicit inclusion of one VM in a vApp will be removed in the next major version.
```

Comment:
Please remove the last part of "will be removed in the next major version", it's enough to state it's deprecated.

Reply:
done